src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/TournamentsController.cs

```diff
@@ -696,7 +698,7 @@ private GameViewModel GetGameViewModelFor(int tournamentId)
             };
         }
 
-        private static List<SelectListItem> BuildTeamSelectList(Dictionary<int, List<TeamTournamentDto>> tournamentTeams, Dictionary<int, SelectListGroup> groups)
+        private static List<SelectListItem> BuildTeamSelectList(IEnumerable<KeyValuePair<int, List<TeamTournamentDto>>> tournamentTeams, IDictionary<int, SelectListGroup> groups)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to remove the nested type argument. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS4017)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Consider using more general type 'System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<int, System.Collections.Generic.List<VolleyManagement.Domain.TeamsAggregate.TeamTournamentDto>>>' instead of 'System.Collections.Generic.IDictionary<int, System.Collections.Generic.List<VolleyManagement.Domain.TeamsAggregate.TeamTournamentDto>>'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3242)